acceptance/chaos_test.go

```diff
@@ -85,7 +84,7 @@ func TestChaos(t *testing.T) {
 					// These originate from DistSender when, for example, the
 					// leader is down. With more realistic retry options, we
 					// should probably not see them.
-					if _, ok := err.(*roachpb.SendError); ok || testutils.IsError(err, rpc.ErrShutdown.Error()) {
+					if _, ok := err.(*roachpb.SendError); ok || testutils.IsError(err, rpc.ErrShutdown.Error()) || testutils.IsError(err, "client is unhealthy") {
```

Comment:
Perhaps `rpcSender.Send` should return a `SendError` if the client is unhealthy.